### PR TITLE
FLU: convert objects/computers

### PR DESCRIPTION
**Pull request by ad-daniel**
**Description**
Ditto.

### CODE REVIEW

File: projects/objects/computers/protos/ComputerMouse.proto [hunk 1]
```diff
@@ -13,63 +13,63 @@ PROTO ComputerMouse [
...
+DEF TOPMOUSE Shape {
```
Comment (lukicdarkoo) [inline projects/objects/computers/protos/ComputerMouse.proto:38]: There is a bug in the script, the indentation doesn't work properly for certain nodes

**Review by lukicdarkoo — CHANGES_REQUESTED** [review]
Thank you, it looks good, just check the indentation

File: projects/objects/computers/protos/ComputerMouse.proto (hunk 1 above)
Comment (ad-daniel) [reply projects/objects/computers/protos/ComputerMouse.proto]: Thanks, didn't notice. I checked the others as well and it appears to be the only case.

**Review by lukicdarkoo — APPROVED** [review]
Thank you

**Comment by omichel** [comment]
I believe we can merge it as the test failures are not related with this PR.

**Pull request merged**